internal/pkg/term/selector/selector.go

```diff
@@ -467,6 +498,14 @@ func (s *WorkspaceSelect) retrieveWorkspaceJobs() ([]string, error) {
 	return localJobNames, nil
 }
 
+func (s *WorkspaceSelect) retrieveWorkspaceWorkloads() ([]string, error) {
+	localWlNames, err := s.ws.WorkloadNames()
+	if err != nil {
+		return nil, err
```

Comment:
nit: does this need to be wrapped?

Reply:
No, we wrap it in the public `Workload()` method. This is the same as the other private `retrieveWorkspaceJobs` and `retrieveWorkspaceServices` methods.